ui/components/app/flask/experimental-area/index.scss

```
@@ -0,0 +1,44 @@
+.experimental-area {
+  color: $flask-purple;
+  margin: auto;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+
+  .logo {
+    padding: 16px;
+    line-height: 0.625rem;
+    font-family: monospace;
+  }
+
+  .experimental-text {
+    padding: 16px;
+    font-family: monospace;
+    margin: auto;
+    line-height: 16px;
+  }
```

This uses a design system sass variable, and adds more space below the experimental text
```suggestion
  .experimental-text {
    padding: 16px 8px 0;
    font-family: monospace;
    font-size: $font-size-h5;
    margin: auto;
    line-height: .875em;
    margin-bottom: 32px;
  }
```

---

We could also use a smaller font-size here to reduce the likelihood of the text breaking because of window size. e.g. `font-size: $font-size-h6;`

---

Yes, please. I knew it will break in small screens, maybe adding a `min-width` will make it safer? Not fan of constraining the layout, tho.